app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettings.kt

```diff
@@ -143,18 +142,11 @@ private fun WallpaperSnackbar(
         },
         action = {
             TextButton(
-                modifier = Modifier.padding(start = 16.dp, end = 16.dp, top = 8.dp, bottom = 8.dp),
+                text = stringResource(R.string.wallpaper_updated_snackbar_action),
                 onClick = onViewWallpaper,
-            ) {
-                Text(
-                    text = stringResource(R.string.wallpaper_updated_snackbar_action).uppercase(
-                        Locale.getDefault()
-                    ),
-                    color = FirefoxTheme.colors.textOnColorPrimary,
-                    fontFamily = FontFamily(Font(R.font.metropolis_medium)),
-                    fontSize = 14.sp,
-                )
-            }
+                modifier = Modifier.padding(all = 8.dp),
```

Comment:
Any reasoning we are dropping the 16dp start/end?

Reply:
I will look to realign this in https://github.com/mozilla-mobile/fenix/issues/23221 with the Figma designs.

Reply:
It looked like too much padding was there (based on eyeballing the symmetry)
![image](https://user-images.githubusercontent.com/87384386/170153393-84c3a557-b98b-4583-af88-07be4e0ed69e.png)